src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,104 @@
+
+For details, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#upgrade-a-cluster).
+
+### Disaster recovery
+
+When cluster virtualization is enabled, the default scope of [backup]({% link {{ page.version.version }}/backup.md %}) and [restore]({% link {{ page.version.version }}/restore.md %}) commands is the virtual cluster. This means that:
+
+- By default, backups taken from a virtual cluster contain only data for that virtual cluster, not the system interface.
+- To back up the entire cluster, you must connect to the system interface to run the backup and include the `INCLUDE_ALL_SECONDARY_TENANTS` flag.
+- A backup of a virtual cluster can be restored as a virtual cluster in any storage cluster with cluster virtualization enabled.
+
+For more details about backing up and restoring a cluster with cluster virtualization enabled, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#disaster-recovery).
+
+{{site.data.alerts.callout_success}}
+For details about configuring and using Physical Cluster Replication for disaster recovery, refer to [Physical Cluster Replication Overview]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}).
+{{site.data.alerts.end}}
+
+### Observability
+
+When cluster virtualization is enabled, cluster log messages and metrics are scoped to the virtual cluster or to the system interface, and are labeled accordingly.
+
+When connected to a virtual cluster, most pages and views are scoped to the virtual cluster. By default the DB Console displays only metrics about that virtual cluster, and excludes metrics for other virtual clusters and the system interface. To allow the DB Console to display system-level metrics from within a virtual cluster, you can grant the virtual cluster the `can_view_node_info` permission.
+
+Some pages and views are only viewable from the system interface, including those pertaining to overall cluster health.
+
+For details, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#observability)
+
+### SQL API
+
+When cluster virtualization is enabled, certain low-level SQL APIs (TODO: Which ones) are accessible only by those users with the `admin` role on the system interface, and not from virtual clusters.
+
+### Span Config Bounds
```

Comment:
Maybe this is covered in more detail later in the review, but I'm not following this section.  There also doesn't seem to be a way to set span configs in our docs.  Perhaps @arulajmani can help clarify this section.